.travis.yml

```diff
@@ -27,4 +27,6 @@ install:
     - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update symfony/symfony=$SYMFONY_VERSION; fi
     - composer update --no-interaction $COMPOSER_FLAGS
 
-script: phpunit --coverage-text --colors
+script:
+    - composer validate --no-check-lock --strict
```

Comment:
``--no-check-lock`` is here because we don't commit the lock file, and so it is useless to validate that it is in sync.
``--strict`` is here to forbid warnings as well.